database/src/schema/account.rs

```diff
@@ -111,6 +111,22 @@ impl Account {
         .map(|_| ())
     }
 
+    // Delete an account
+    //
+    // Do expose this to be used outside of testing. When we support account deletion,
```

Comment:
```suggestion
    // Don't expose this to be used outside of testing. When we support account deletion,
```
Right? What happened here?